common/flatpak-dir.c

```diff
@@ -7700,6 +7841,31 @@ flatpak_dir_create_system_child_repo (FlatpakDir   *self,
   return flatpak_dir_create_child_repo (self, cache_dir, file_lock, optional_commit, error);
 }
 
+static void
+flatpak_dir_cancel_pull (FlatpakDir    *self,
```

Comment:
This to me reads like it will just call CancelPull. Perhaps name it `flatpak_dir_unmount_and_cancel_pull`?